freshmaker/events.py

```diff
@@ -486,3 +486,10 @@ def __init__(self, msg_id, advisory, container_images,
     @property
     def search_key(self):
         return str(self.advisory.errata_id)
+
+
+class FlatpakApplicationManualBuildEvent(ManualRebuildWithAdvisoryEvent):
+    """
+    Event triggered when Release Driver requests manual rebuild
+    OR when manual rebuild of a flatpak is requested by person.
```

Comment:
Flatpak advisories won't be handled by release-driver, it's better to update this doc, actually release-driver requests are just manual requests which is not very special, it worth nothing to mention release-driver in any of the manual rebuild event.

Reply:
I removed release-driver mention from the docstring.